graylog2-rest-client/src/main/java/org/graylog2/restclient/models/dashboards/widgets/DashboardWidget.java

```diff
@@ -236,6 +238,21 @@ public static DashboardWidget factory(Dashboard dashboard, DashboardWidgetRespon
                         w.creatorUserId
                 );
                 break;
+            case STACKED_CHART:
+                widget = new StackedChartWidget(
+                        dashboard,
+                        w.id,
+                        w.description,
+                        w.cacheTime,
+                        TimeRange.factory((Map<String, Object>) w.config.get("timerange")),
+                        (w.config.containsKey("stream_id") ? (String) w.config.get("stream_id") : null),
```

Comment:
Makes sense, will also replace how we get the `stream_id` to create other widget instances, as they all look like that.